File: Code/Client/src/app/models/admin/Accesorio.ts
import { Basemodel } from "../base/Basemodel";

export class Accesorio  extends Basemodel{
  nombre: string | null;
  modelo: string | null;
  tipo: string | null;
  descripcion?: string | null;
  codigo_imt: string | null;
  precio?: number|null;
  url_data_sheet?: string|null;
  nombreEquipoAsociado?: string | null;

  constructor() {
    super();
    this.nombre = null;
    this.modelo = null;
    this.tipo = null;
    this.descripcion = null;
    this.codigo_imt = null;
    this.precio = null;
    this.url_data_sheet = null;
    this.nombreEquipoAsociado = null;
  }

}

File: Code/Server/src/Core/Entities/Accesorio.cs
namespace IMT_Reservas.Server.Core.Entities;

public class Accesorio : Entity
{
    public string? Nombre { get; set; }
    public string? Descripcion { get; set; }
    public string? Modelo { get; set; }
    public string? UrlDataSheet { get; set; }
    public decimal? Precio { get; set; }
    public int IdEquipo { get; set; }
    public string? Tipo { get; set; }
    public bool EstadoEliminado { get; set; }
}
